src/art/Mwheel/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=h, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Special+Elite&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>THE WHEEL</title>
+</head>
+<body>
+    <h1>MOTION WHEEL</h1>
+  <img src="image/car.png" alt="wheel-image">
```

Comment:
Please provide the license to the image. Images are normally under copyright, so we need to make sure that we don't violate copyright.

Reply:
It's a copyright free image took from flaticon

Reply:
According to [their website](https://support.flaticon.com/s/article/Is-Flaticon-for-free?language=en_US) you need to add an attribution line.

Please point to the code where you do that. I haven't seen it, but maybe I've overlooked it. Thank you.